app/src/Command/ExportListCommandsCommand.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ExportListCommandsCommand extends Command
+{
+    protected static $defaultName = 'app:list-commands';
+
+    protected function configure(): void
+    {
+        $this
+          ->setDescription('List all available Symfony commands with their documentation.')
+          ->setHelp('This command lists all available Symfony commands and their descriptions.');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $commands = $this->getApplication()->all();
+        $commandList = [];
+
+        foreach ($commands as $name => $command) {
```

Comment:
We will need the full detail of argument and option.

End result should have same information we get from 

`bin/console {commandName} --help`

![image](https://github.com/mpoiriert/draw/assets/4175616/25f9eb0d-906f-46c0-8bd8-a0c1db6abb6a)